openedx/adg/lms/our_team/models.py

```diff
@@ -27,18 +27,32 @@ def board_of_trustees(self):
         return super().get_queryset().filter(member_type=OurTeamMember.BOARD_MEMBER)
 
 
+#  pylint: disable=no-member
 class OurTeamMember(models.Model):
     """
     Model for team members and board of trustees in about page
     """
 
-    name = models.CharField(verbose_name=_('Name'), max_length=150, )
-    designation = models.CharField(verbose_name=_('Designation'), max_length=150, )
+    name = models.CharField(
+        verbose_name=_('Name'),
+        max_length=70,
+        help_text=_('Add a name of max {char_limit} characters').format(char_limit=MAX_NAME_LENGTH),
```

Comment:
lets use f string here and in the help text of designation and description as well

Reply:
Cannot use fstrings with translations.
https://code.djangoproject.com/ticket/30109